packages/tasks/src/model-libraries.ts

```diff
@@ -95,6 +95,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.audiocraft,
 		filter: false,
 	},
+	audioseal: {
+		prettyLabel: "Audioseal",
```

Comment:
```suggestion
		prettyLabel: "AudioSeal",
```

I think that's what they use in their repo.